### ISSUE TITLE
Error while building meteor app in polymer-flow-router boilerplate

### ISSUE THREAD

**Issue by itsashis4u** [meteor-useraccounts/boilerplates#24]
I followed the steps in the README
```
cd yourproject
mkdir packages
cd packages
git clone https://github.com/ThaumRystra/meteor-vulcanize.git
cd meteor-vulcanize
git checkout fix-meteor-1.2
cd ../..
meteor update
meteor
```
But when I run `meteor` command, it says `run: You're not in a Meteor project directory.`

I guess the `.meteor` folder doesn't have the `packages` file. How can I generate the `packages` file?

**Comment by itsashis4u** [comment]
The `.gitignore` file maybe creating problems

**Comment by splendido** [comment]
Good catch!

This is what I've just tried:

```
git clone https://github.com/meteor-useraccounts/boilerplates.git
cd boilerplates/polymer-flow-router
mkdir packages && cd packages
git clone https://github.com/ThaumRystra/meteor-vulcanize.git
cd meteor-vulcanize
git checkout fix-meteor-1.2
cd ../..
bower install
meteor update
meteor
```

and it worked.

Thanks!

**Comment by itsashis4u** [comment]
Thanks for resolving it promptly :+1:
